search_events_app/services/filters/country_filter.py

```diff
@@ -7,11 +7,11 @@
 class CountryFilter(Filter):
 
     def apply_filter(self, request):
-        alpha_2_code = request.GET.get('country') if request.GET.get('country') else None
+        country_name = request.GET.get('country') if request.GET.get('country') else None
```

Comment:
`request.GET.get('country', None)` or `request.GET.get('country') or None`
are 2 options that maybe you could use too